yarn-project/test-wallet/src/wallet/test_wallet.ts

```diff
@@ -16,7 +16,7 @@ import {
 import { DefaultMultiCallEntrypoint } from '@aztec/entrypoints/multicall';
 import type { ExecutionPayload } from '@aztec/entrypoints/payload';
 import { Fq, Fr } from '@aztec/foundation/fields';
-import type { PXE } from '@aztec/pxe';
+import type { PXE } from '@aztec/pxe/server';
```

Comment:
Even if just a type import, prefer /client/lazy ones to be safe when these can be imported from the browser